--- rubygem-libxml-ruby-1.1.3/metadata.licenses~	2012-02-16 06:20:35.938113880 +0100
+++ rubygem-libxml-ruby-1.1.3/metadata	2012-02-16 06:20:49.495129412 +0100
@@ -16,6 +16,7 @@ dependencies: []
 description: The Libxml-Ruby project provides Ruby language bindings for the GNOME Libxml2 XML toolkit. It is free software, released under the MIT License. Libxml-ruby's primary advantage over REXML is performance - if speed  is your need, these are good libraries to consider, as demonstrated by the informal benchmark below.
 email: libxml-devel@rubyforge.org
 executables: []
+licenses: []
 
 extensions: 
 - ext/libxml/extconf.rb
